app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetingSlice';
+
+const Greeting = () => {
+  const greeting = useSelector((state) => state.greetings.value);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, [dispatch]);
+
+  return (
+    <div className='greeting_div'>
+      <h2>Random Greeting:</h2>
+      <p className='greeting'>{greeting}</p>
```

Comment:
- [ ] You did an awesome job, by implementing the back-end correctly, API alone is working fine and your UI is rendering correctly, the only problem that I got is your Greetings are not rendering on the UI and further when I checked the `greeting` coming from the `redux` store and I found that I got nothing. So the problem is coming from your `Redux` implementation, which needs to be addressed before asking for the next review. Kindly fix the issue re-check your work, make sure that everything is working smoothly.

![Screenshot from 2024-01-15 15-45-33](https://github.com/MozamelJawad/hello-rails-react/assets/66476329/f1a3da89-d5ba-416d-927f-f3d642bfc591)

Reply:
Thanks for your feedback.
This is all working on my computer 
![Screenshot 2024-01-15 145717](https://github.com/MozamelJawad/hello-rails-react/assets/14952154/51ebb6b2-9a51-4186-94ed-531183932507)

![Screenshot 2024-01-15 145741](https://github.com/MozamelJawad/hello-rails-react/assets/14952154/6ac8ce85-e803-4298-9d10-ecaf7b36dd58)